packages/form-generator/package.json

```diff
@@ -19,8 +19,8 @@
   "license": "Apache-2.0",
   "dependencies": {
     "@aws-amplify/appsync-modelgen-plugin": "^2.6.0",
-    "@aws-amplify/codegen-ui": "^2.19.4",
-    "@aws-amplify/codegen-ui-react": "^2.19.4",
+    "@aws-amplify/codegen-ui": "^2.20.1",
```

Comment:
Since this is within the semver range of ^2 customers will automatically get this upgrade when running `npm update`. We shouldn't need this change unless there is some particular feature we need to consume in the amplify-backend repo

Reply:
Okay cool, closing it